week1/assignment2/style.css

```
@@ -0,0 +1,247 @@
+@import "reset.css";
+
+* {
+  box-sizing: border-box;
+}
+
+hr {
+  margin: 0;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+}
+
+/* 제목 */
+header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 2.9rem;
+  text-align: center;
+  background-color: lightsalmon;
+  color: white;
+  font-size: 1.6rem;
+  font-weight: 600;
+}
+
+#wrap {
+  min-height: calc(100% - 3rem);
+}
+
+/* 나의 자산 총 정리 부분 */
+#total_budget_section {
+  width: 100%;
+  height: 14rem;
+  padding: 2rem;
+  border-width: 0.1rem 0;
+}
+
+#total_budget_wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 10rem;
+  padding: 1rem;
+  gap: 0.5rem;
+  background-color: peachpuff;
+  border-radius: 1rem;
+  border: solid 0.15rem gray;
+}
+
+#total_budget_title {
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+#total_amount {
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+#todays_history {
+  display: flex;
+  margin-top: 0.2rem;
+  font-size: 1.4rem;
+  gap: 1.2rem;
+}
+
+/* icon 디자인 생각해보기 */
+.icon {
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+#income {
+  display: flex;
+  align-items: center;
+  gap: 0.1rem;
+  color: blue;
+  font-size: 1.3rem;
+}
+
+#expense {
+  display: flex;
+  align-items: center;
+  gap: 0.1rem;
+  color: red;
+  font-size: 1.3rem;
+}
+
```

이런식으로 color만 스타일링이 다르고 겹치는 것이 많은 경우에는 html 요소에서 가져와 보자면
```suggestion
#todays_history > div {
  display: flex;
  align-items: center;
  gap: 0.1rem;
  font-size: 1.3rem;
}

#income {
  color: blue;
}

#expense {
  color: red;
}

```
이렇게 써주면 더 좋겠죠?!